manifests/init.pp

```diff
@@ -56,17 +56,22 @@
 #
 class svckill (
   Array[String]               $ignore       = [],
+  Array[String]               $ignore_defaults = [],
   Array[Stdlib::Absolutepath] $ignore_files = [],
-  Boolean                     $verbose      = true
+  Boolean                     $verbose      = true,
+  Boolean                     $debug        = false
 ){
   include '::svckill::ignore::collector'
-
+  $combined_ignore_list = $ignore + $ignore_defaults
+  if ($debug == true) {
```

Comment:
I didn't want to make a lot of changes to that ruby code, and the point of this debugging line is to debug the hiera based ignore list functionality, not the type/provider. ie, to see if the consolidation of the module data into a single array is including /all/ of the intended ignored resources.

I have a number of issues with how this type/provider work and how they send output, which I didn't want to get into. I mainly left it in precisely to ease debugging.